### PR TITLE
#351 sdkv2 liquidity pools

### PR DESCRIPTION
**Pull request by yornaath**
Working example(hopefully 😅) of the liquidity pools list page implemented using sdk v2.

### CODE REVIEW

**Comment by Robiquet** [comment]
Is it possible to keep pagination like with the previous implementation?

File: package.json [hunk 1]
```diff
@@ -47,8 +53,9 @@
     "mobx": "^6.3.12",
     "mobx-react": "^7.2.1",
     "mobx-react-form": "^3.2.0",
-    "next": "^12.2.5",
+    "next": "^12.3.1",
     "object-hash": "^2.2.0",
+    "parse-duration": "^1.0.2",
```
Comment (Robiquet) [inline package.json:58]: Is this used?

File: tailwind.config.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ module.exports = {
       kanit: ["Kanit"],
       lato: ["Lato"],
       space: ["Space Grotesk"],
+      roboto: ["Roboto Mono"],
```
Comment (Robiquet) [inline tailwind.config.js:59]: Think we already have this configured as `mono`. See:
```      
mono: [
        "Roboto Mono",
```

**Review by Robiquet — COMMENTED** [review]
See comments

File: package.json [hunk 3]
```diff
@@ -25,16 +24,23 @@
...
+    "@tanstack/react-query": "^4.13.0",
+    "@zeitgeistpm/avatara-nft-sdk": "^1.1.5",
+    "@zeitgeistpm/avatara-react": "^1.1.5",
     "@zeitgeistpm/avatara-util": "^1.0.0-alpha.0",
-    "@zeitgeistpm/sdk": "0.9.0",
+    "@zeitgeistpm/sdk": "^0.9.1",
+    "@zeitgeistpm/sdk-next": "npm:@zeitgeistpm/sdk@2.1.10",
+    "@zeitgeistpm/utility": "^2.1.4",
     "axios": "^0.21.4",
+    "bignumber.js": "9.1.0",
```
[the remark this answers is not in the document]
Comment (yornaath) [reply package.json:43]: We can, but according to this post bignumber is better for financial applications. There is a reference to where decimal.js seems to produce the wrong output?

https://dev.to/fvictorio/a-comparison-of-bignumber-libraries-in-javascript-2gc5#fn1

File: package.json (hunk 1 above)
Comment (yornaath) [reply package.json:58]: Nope!

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Robiquet) [reply package.json:43]: I can't reproduce that:
``console.log(new Decimal("22222222222222222222").div(2).toFixed(0));``

logs:
`11111111111111111111`

**Comment by Robiquet** [comment]
Probably this would be a good link 'Learn' button https://docs.zeitgeist.pm/docs/learn/liquidity

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (yornaath) [reply package.json:43]: ok yeah, shouldnt be a problem to refactor to use decimal

**Comment by yornaath** [comment]
@saboonikhil on hold since we need the total liquidity for all pools gql schema.

**Comment by yornaath** [comment]
Moved to #396

**Pull request closed without merging**